admin_guide/pruning_resources.adoc

```diff
@@ -272,3 +274,49 @@ $ oadm prune images --keep-tag-revisions=3 --keep-younger-than=60m --confirm
 
 $ oadm prune images --prune-over-size-limit --confirm
 ----
+
+[[image-pruning-problems]]
+=== Image pruning problems
+If your images keep pilling up and prune command removes just a small portion
+of what you'd expect, make sure you understand xref:image-prune-conditions[the
+conditions] that must apply for an image to be considered a candidate for pruning.
+
+Especially, ensure that images you want removed occur at higher positions in
+each
+xref:../architecture/core_concepts/builds_and_image_streams.adoc#image-stream-tag[tag
+history] than your chosen tag revisions threshold. For example, consider an old
+and obsolete image named `sha:abz`. By running following command in particular
+namespace `N` where the image is tagged:
+
+----
+$ image_name="sha:abz"
+$ oc get is -n N -o go-template='{{range $isi, $is := .items}}{{range $ti, $tag := $is.status.tags}}'\
+  '{{range $ii, $item := $tag.items}}{{if eq $item.image "'"${image_name}"\
+  $'"}}{{$is.metadata.name}}:{{$tag.tag}} at position {{$ii}} out of {{len $tag.items}}\n'\
+  '{{end}}{{end}}{{end}}{{end}}'
+myapp:v2 at position 4 out of 5
+myapp:v2.1 at position 2 out of 2
+myapp:v2.1-may-2016 at position 0 out of 1
+----
+
+you find out that image is tagged three times in a single _istag_ named `myapp`.
+The image won't ever be pruned using standard prunning options because it
+occurs at position 0 in a history of `myapp:v2.1-may-2016` tag. For an image to
+be prunable, the admin would have to either:
+
+1. Specify `--keep-tag-revisions=0` to `oadm prune images` command, which is
+very dangerous and definitely **not** recommended.
+2. Delete all the
+xref:../architecture/core_concepts/builds_and_image_streams.adoc#image-stream-tag[_istags_]
+where the position is below the revision threshold. In this case _istags_
+`myapp:v2.1` and `myapp:v2.1-may-2016`.
+3. Move the image further in the history by performing new builds to the same
```

Comment:
in the history, either running new builds pushing to the same istag, or tagging other image.  Unfortunately, this might not always be desirable for old release tags.